pypom/page.py

```diff
@@ -8,18 +8,48 @@
 class Page(WebView):
 
     _url = None
+    """
+    A string which when appended to ``base_url``
+    indicates the url to the given page.
```

Comment:
We're not appending, we're formatting the string and passing the base URL as a keyword argument.